### ISSUE TITLE
"Met Exception" when training model

### ISSUE THREAD

**Issue by mujtabatirmizi** [Microsoft/LightGBM#661]
Hi, I was using the LightGBM repo to train a relevance model using lambdarank. I've trained a couple models using the same config. For some reason after I increased the size of my training data I'm getting an error: 

"Met Exceptions:
Label excel 947753232"

I couldn't seem to find much documentation on what that exception exactly meant. I was wondering whether the issue was with the size of the training data or with the actual values for the label column? Any clarification would be helpful. Also let me know if there is any other info I should provide about the issue.

**Comment by guolinke** [comment]
@mujtabatirmizi 
I think you are using wrong label. 
For the ranking, the label is ```int``` type, and represent the relevant. e.g. 
0: bad
1: good
2. perfect
